polling_stations/apps/uk_geo_utils/geocoders.py

```diff
@@ -52,13 +52,13 @@ def __init__(self, postcode):
             raise AddressBaseNotImportedException('Address Base table is empty')
 
         self.addresses = self.address_model.objects.filter(
-            postcode=self.postcode.with_space)
+            postcode=self.postcode.with_space).order_by('uprn')
```

Comment:
If performance is a problem here (not sure it will be, as only a few number of rows will be returned) then postgres can default to ordering when you make an index https://www.postgresql.org/docs/8.3/static/indexes-ordering.html

Reply:
I'm not too worried about this for the reason you've just mentioned, but lets bear it in mind if we benchmark and need to optimise further. I think reducing the number of round trips to the DB has done the '80' bit of the optimisation we need and this is somewhere in the '20'